<launch>
  <node name="DetectionNode" pkg="ts_detection" type="yolo_detections.py" output="screen"/>
</launch>